flink-runtime/src/main/java/org/apache/flink/runtime/operators/util/TaskConfig.java

```diff
@@ -238,6 +240,19 @@
 
     private static final String SOLUTION_SET_OBJECTS = "itertive.ss.obj";
 
+    public static final ConfigOption<Integer> THROUGHPUT_CALCULATION_PERIOD =
+            ConfigOptions.key("io.throughput.calculationPeriod")
+                    .intType()
+                    .defaultValue(500)
+                    .withDescription("The amount of time between calculation the throughput(ms).");
+
+    public static final ConfigOption<Integer> THROUGHPUT_EMA_LAST_VALUES =
```

Comment:
It is actually not clear where should be both these configurations. THROUGHPUT_CALCULATION_PERIOD relates to streaming and perhaps it is better to move it to StreamConfig but holding them in separate places is also not a good idea. 
So any ideas for the right place for these configs?

Reply:
`taskmanager.memory.network.in-flight-data-adjustment.period` +
`taskmanager.memory.network.in-flight-data-adjustment.samples` +
move to `TaskManagerOptions`?

Exposing configuration also could have been another commit - it would be easier to pull in someone else to verify that the config names/placement make sense.